packages/perspective-viewer/src/html/computed_expression_widget.html

```diff
@@ -0,0 +1,32 @@
+<!--
+
+Copyright (c) 2017, the Perspective Authors.
+
+This file is part of the Perspective library, distributed under the terms of
+the Apache License 2.0.  The full license can be found in the LICENSE file.
+
+-->
+
+<template id="perspective-computed-expression-widget">
+    <div class="psp-computed-expression-widget__body psp-computed-expression-widget__container">
+        <div class="psp-computed-expression-widget__container">
+            <div class="psp-computed-expression-widget__action">
+                <span class="psp-icon psp-icon__close" id="psp-computed-expression-widget-close" aria-label="Close Computed Expression Widget"></span>
+                <span id="psp-computed-expression-widget-title">New Column</span>
+            </div>
+            <div class="psp-computed-expression-widget__content psp-computed-expression-widget__content--nomargin">
```

Comment:
I am aware that these class names are not exposed in the light DOM and are not currently styleable;  also that this project has generally no consistently named classes.  @zemeolotu and I discussed the following scheme when building out the theme support for `perspective-viewer` and `perspective-workspace`:
 
* `--${Perspective Component Name}--${Actual CSS Property name}` (for public style variables)

* `--${Column Types}--${Perspective Component Name}--${Actual CSS Property name}` (for public style variables specific to a style)

and

* `${Perspective Component Name}--${Actual CSS Property name}` (for internal style variables)